src/neuroconv/datainterfaces/ecephys/spikeglx/spikeglx_utils.py

```diff
@@ -107,7 +107,7 @@ def get_device_metadata(meta) -> dict:
             "1030": "NP1.0 NHP",
         }
         probe_type = str(meta["imDatPrb_type"])
-        probe_type_description = probe_type_to_probe_description[probe_type]
+        probe_type_description = probe_type_to_probe_description.get(probe_type, "")
```

Comment:
```suggestion
        probe_type_description = probe_type_to_probe_description.get(probe_type, "Unknown SpikeGLX probe type.")
```

How about this? Missing or empty descriptions are a best practice violation, and this at least indicates that we tried to look it up

Reply:
Sounds better! I will try to extract this from probeinterface as soon as I can : )